python_examples/age_stratified_example/simulation_flow_age_spatial.py

```diff
@@ -0,0 +1,116 @@
+#
+# Example simulation script with spatial data output and visualisation
+# Currently running with Kit's gibraltar set-up
```

Comment:
Yes I think let's move this to a sub directory within the Gibraltar simulation, and use the same input and parameters files